Xamarin.PropertyEditing.Mac/PropertyEditorPanel.cs

```diff
@@ -197,16 +197,21 @@ private void Initialize ()
 				NSLayoutConstraint.Create (this.propertyList, NSLayoutAttribute.Width, NSLayoutRelation.Equal, this, NSLayoutAttribute.Width, 1, 0),
 			});
 
-			ViewDidChangeEffectiveAppearance ();
+			UpdateResourceProvider ();
 		}
 
-		private void UpdateResourceProvider()
+		public override void ViewDidChangeEffectiveAppearance ()
+		{
+			base.ViewDidChangeEffectiveAppearance ();
+
+			UpdateResourceProvider ();
```

Comment:
Should this method be renamed `AppearanceChanged` for consistency with every other implementation?

Reply:
@alanmcgovern I did consider it, but currently it specifically only updates the respective ResourceProvider and not any other Appearance values for specific control. If you really want it it changed, I can do that.

Reply:
I suppose i'm just looking for a consistent set of `sealed override ViewDidChangeEffectiveAppearance` and `void UpdateAppearance` methods.

Let's put in a pin in this for now as i just did a quick code review of the rest of the codebase and have some concerns. I'll file an issue with my thoughts!